.github/workflows/go.yml

```diff
@@ -0,0 +1,27 @@
+name: Go
+on: [push]
+jobs:
+
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Set up Go 1.12
+      uses: actions/setup-go@v1
+      with:
+        go-version: 1.12
```

Comment:
I'm unfamiliar with actions/setup-go, but is there any reason you chose Go 1.12 instead of Go 1.13?

Reply:
Nope, that was just the default for the example. Updated.